projects/components/src/tabs/content/tab/tab.component.ts

```diff
@@ -10,4 +10,7 @@ import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../../../content/content
 export class TabComponent extends ContentHolder {
   @Input()
   public label!: string;
+
+  @Input()
+  public subText?: string;
```

Comment:
should we call it `summaryText` ? Or what was the other word you mentioned?  `chipText` ?

Reply:
Is this actually subtext? From the screenshot it looks more like a badge.

Reply:
Yeah, The name was open for suggestion. Thanks!
I have fixed this